src/packages/cli/src/bin.ts

```diff
@@ -197,6 +205,40 @@ async function main(): Promise<number> {
   return 0
 }
 
+/**
+ * Get a unique identifier for the project by hashing
+ * the directory with `schema.prisma`
+ */
+async function getProjectHash(): Promise<string> {
```

Comment:
Is this the best spot for utilities @timsuchanek?

Reply:
This could be moved to `@prisma/sdk`

Reply:
Maybe here https://github.com/prisma/prisma/blob/master/src/packages/sdk/src/cli/utils.ts